flink-runtime/src/main/scala/org/apache/flink/runtime/taskmanager/TaskManager.scala

```diff
@@ -585,7 +585,7 @@ class TaskManager(
               config.getMaxRegistrationPause().toMilliseconds,
               TimeUnit.MILLISECONDS))
 
-            // schedule (with our timeout s delay) a check triggers a new registration
+            // schedule (with our timeout's delay) a check triggers a new registration
```

Comment:
"triggers" -> "to trigger"? Or rewrite like "schedule a check to trigger a new registration attempt if not registered by the timeout"?